etc/launch-tae-CN-tae-DN/cn.toml

```diff
@@ -3,8 +3,6 @@ data-dir = "./mo-data"
 
 [log]
 level = "info"
-format = "console"
```

Comment:
PS: `format` default  val = "json"

Reply:
OK， this seems to be a useful thing to expose?   but if format is console, not json, will this affect our system table?

Reply:
It will NOT.